schemas/NotificationPermissionChanged.json

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "object",
+  "description": "Triggered when the user changes the notification permission status.",
+  "properties": {
+    "eventName": {
+      "enum": ["NotificationPermissionChanged"]
```

Comment:
Would it make sense to generalise this event a little but as `PermissionChanged` with a property that indicates which permission? One day we might like to track permissions such as Mic/Camera/Contacts etc.

Reply:
Sounds like a great idea!